src/po/Makefile

```diff
@@ -313,8 +313,9 @@ $(LANGUAGES):
 # For translations of plug-ins
 #######
 
+POT_PLUGPACKAGE_PATH = ${PWD}
```

Comment:
Any reason to use `${}` instead of `$()`? (Both are the same, though.)

Reply:
As I understand it, the *BSD world prefers curly braces. And the GNU world also, in principle, uses them for variables.
But in this makefile, parentheses are used everywhere, so I fixed it. A uniform style must be observed.